package.json

```diff
@@ -7,6 +7,9 @@
     "@algolia/autocomplete-js": "1.7.1",
     "@algolia/autocomplete-theme-classic": "1.7.1",
     "@apollo/client": "^3.6.9",
+    "@dnd-kit/core": "^6.0.8",
```

Comment:
Is the old `react-beautiful-dnd` still being used elsewhere?  if not can you yarn remove it?

Reply:
Yeah it's still being used for CragTable. After this, I'll switch CragTable over and we remove `react-beautiful-dnd`.

Reply:
This crag table?  I don't think we use this component anywhere. 
https://github.com/OpenBeta/open-tacos/blob/develop/src/components/crag/cragTable.tsx

Reply:
Seems like it's used in https://openbeta.io/finder. I didn't realize we had a page like that. Doesn't seem fully functional, but seems like a useful page. What do you think we should do with it? Fix it up? Deprecate the page?

Reply:
I believe there are 2 crag table components. Anyway, I'm merging this now and we can clean it up in a follow up PR.

Reply:
Follow up here: https://github.com/OpenBeta/open-tacos/pull/839